### ISSUE TITLE
the ask file saver and opener in browser is out of screen 

### ISSUE THREAD

**Issue by Manvendra-div** [leftwm/leftwm#680]
**Describe the bug**
the ask file saver and opener in browser is out of screen _maybe its not the bug,i don't know_

**To Reproduce**
1. Download any file from browser or upload any file
2. the filedialog is outofscreen

**Screenshots**
![ArcoLinux-2022-01-27-1643260789_screenshot_1366x768](https://user-images.githubusercontent.com/67854646/151297274-14f09ea7-5c43-4499-b29d-800c7f24895b.jpg)

**Environment:**
 - OS/Distro: Arcolinux
 - Output of `leftwm-check`

**Comment by cesarFuhr** [comment]
@Manvendra-div, can you give us the result of the `leftwm-check` command?

**Comment by VuiMuich** [comment]
Also (especially when you are on the latest git version) the `xprop` of the dialog window might be helpful. Thanks.

**Comment by Manvendra-div** [comment]
:: LeftWM version: 0.3.0
:: LeftWM git hash: 52e340b
:: Loading configuration . . .
    -> Configuration loaded OK 
:: Checking keybinds . . .
    -> All keybinds OK
:: Checking environment . . .
    -> Environment OK 
:: Checking theme . . .
    -> Theme OK

**Comment by Manvendra-div** [comment]
output of `xprop`

```
_NET_WM_DESKTOP(CARDINAL) = 0
WM_STATE(WM_STATE):
		window state: Normal
		icon window: <field not available>
WM_TRANSIENT_FOR(WINDOW): window id # 0xe00003
_NET_WM_STATE(ATOM) = _NET_WM_STATE_MODAL
WM_HINTS(WM_HINTS):
		Client accepts input or input focus: True
		Initial state is Normal State.
		window id # of group leader: 0x5600001
_GTK_THEME_VARIANT(UTF8_STRING) = 
XdndAware(ATOM) = BITMAP
WM_WINDOW_ROLE(STRING) = "GtkFileChooserDialog"
_NET_WM_WINDOW_TYPE(ATOM) = _NET_WM_WINDOW_TYPE_DIALOG
_NET_WM_SYNC_REQUEST_COUNTER(CARDINAL) = 90179496, 90179497
_NET_WM_USER_TIME(CARDINAL) = 10650761
_NET_WM_USER_TIME_WINDOW(WINDOW): window id # 0x56007a7
WM_CLIENT_LEADER(WINDOW): window id # 0x5600001
_NET_WM_PID(CARDINAL) = 1427
WM_LOCALE_NAME(STRING) = "en_IN"
WM_CLIENT_MACHINE(STRING) = "arcolinux"
WM_NORMAL_HINTS(WM_SIZE_HINTS):
		program specified minimum size: 546 by 412
		program specified base size: 0 by 0
		window gravity: NorthWest
WM_PROTOCOLS(ATOM): protocols  WM_DELETE_WINDOW, WM_TAKE_FOCUS, _NET_WM_PING, _NET_WM_SYNC_REQUEST
WM_CLASS(STRING) = "xdg-desktop-portal-gtk", "Xdg-desktop-portal-gtk"
WM_ICON_NAME(STRING) = "All Files"
_NET_WM_ICON_NAME(UTF8_STRING) = "All Files"
WM_NAME(STRING) = "All Files"
_NET_WM_NAME(UTF8_STRING) = "All Files"

```

**Comment by VuiMuich** [comment]
May I ask for one further output to gather? Please run `xwininfo` on the popup.
I think a fix for this might be to check if the corners are out of bounds (tagging #683) and if needed resort to `program specified minimum size`.

**Comment by cesarFuhr** [comment]
I asked for `leftwm-check` because I think #683 solution will fix it. Right now he is running a version that does not include the fix.

**Comment by VuiMuich** [comment]
Yeah, thought you where aiming for that.
But #683 did no resizing, right?

**Comment by cesarFuhr** [comment]
It doesn't... but I would not try to fix it before testing it with the fix included in the build.

**Comment by hertg** [comment]
I'm optimistically assuming this was fixed with #683
Please reopen if that's not the case.
